docs/Using-EthSigner/Using-EthSigner.md

```diff
@@ -15,6 +16,55 @@ by EthSigner.
+
+**Parameters**
+
+Transaction object for private transactions: 
+
+| Key             | Type                | Required/Optional                  | Value                                                                                                                         |
+|-----------------|--:-:----------------|----------------------------------- |-------------------------------------------------------------------------------------------------------------------------------|
+| **from**        | Data, 20&nbsp;bytes | Required                           | Address of the sender. Must be the address of the keystore account.                                                                                                        |
+| **to**          | Data, 20&nbsp;bytes | Not required for contract creation | `null` for contract creation transaction. Contract address for contract invocation transactions.                                                           |
+| **gas**         | Quantity            | Optional                           | Gas provided by the sender. Default is `90000`.                                                                               |
+| **gasPrice**    | Quantity            | Optional                           | Gas price provided by the sender in Wei. Default is `0`.                                                                      |
+| **nonce**       | Quantity            | Optional                           | Number of transactions sent from the `from` account before this one.                                                                    |
+| **data**        | Quantity            | Optional                           | Compiled contract code or hash of the invoked method signature and encoded parameters.                                        |
+| **privateFrom** | Data, 20&nbsp;bytes | Required                           | Orion address of the sender                                                                                                         |
+| **privateFor**  | Array of data       | Required                           | Orion addresses of recipients                                                                                                       |
+| **restriction** | String              | Required                           | Must be [`restricted`](https://docs.pantheon.pegasys.tech/en/stable/Privacy/Privacy-Overview/#private-transaction-attributes) |
+
+!!! tip
+    Submitting a transaction with the same nonce as a pending transaction and a higher gas price replaces 
+    the pending transaction with the new one. If not attempting to replace a pending transaction, do not 
+    include the `nonce` in the private transaction object and nonce management is handled automatically. 
+
+!!! note
+    If a `value` is included in the transaction object, an error is returned.  Ether transfers cannot 
```

Comment:
It's only an error if the value is included and it isn't zero

Reply:
Fixed